services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -938,18 +940,19 @@ class HankeServiceITests : DatabaseTest() {
 
         @Test
         fun `creates hankealueet from the application areas`() {
-            val inputApplication = AlluDataFactory.cableReportWithoutHanke()
-            assertThat(inputApplication.applicationData.areas).isNotEmpty
+            val inputApplication =
+                AlluDataFactory.cableReportWithoutHanke(
+                    AlluDataFactory.createCableReportApplicationData(areas = null)
+                        .withArea(name = "Area", geometry = GeometriaFactory.secondPolygon)
+                )
 
             val application = hankeService.generateHankeWithApplication(inputApplication, USER_NAME)
 
             val hanke = hankeService.loadHanke(application.hankeTunnus)!!
-            hanke.alueet.forEach { hankealue ->
-                val features = hankealue.geometriat?.featureCollection?.features
-                assertThat(features).hasSize(1)
-                val polygon = features!![0].geometry as Polygon
-                assertThat(polygon.coordinates)
-                    .isEqualTo(inputApplication.applicationData.areas!![0].geometry.coordinates)
+            assertThat(hanke.alueet).hasSize(1)
+            assertk.assertThat(hanke.alueet).first().all {
```

Comment:
Why `assertk.assertThat` instead of just `assertThat`?

Reply:
In this file `assertThat` refers to AssertJ, and I couldn't find a clean way to do this in AssertJ.

Reply:
Good point. It might be possible to get rid of AssertJ in this test class but maybe no need to do that now.